src/mlpack/core/data/split_data.hpp

```diff
@@ -14,6 +14,7 @@
 #define MLPACK_CORE_DATA_SPLIT_DATA_HPP
 
 #include <mlpack/prereqs.hpp>
+#include <deque>
```

Comment:
I'm not sure if this include is necessary; let me know if I overlooked something.

Reply:
Not neccesary, correct!